README.md

```diff
@@ -1 +1,12 @@
-# hw01
\ No newline at end of file
+# Homework 01
```

Comment:
I really appreciate how detailed your comment was and how you went over each step that caused you trouble because it was really informative for me. I had a similar issue with not staging the readme file and getting the message that everything was up to date, so I am glad that we both figured this out. And your process of uploading the image was really informative and speaks to how important it is to have everything aligned on the file path